Extension/src/LanguageServer/client.ts

```diff
@@ -868,6 +877,86 @@ export class DefaultClient implements Client {
+                    return;
+                case 'darwin':
+                    vscode.commands.executeCommand('vscode.open', "https://go.microsoft.com/fwlink/?linkid=2217706");
+                    return;
+                default: // Linux
+                    vscode.commands.executeCommand('vscode.open', "https://go.microsoft.com/fwlink/?linkid=2217615");
+                    return;
+            }
+        } else if (index === paths.length - 3) {
+            const result: vscode.Uri[] | undefined = await vscode.window.showOpenDialog();
+            if (result !== undefined && result.length > 0) {
+                util.addTrustedCompiler(compilerPaths, result[0].fsPath);
+                settings.defaultCompiler = result[0].fsPath;
+                compilerDefaults = await this.requestCompiler(compilerPaths);
+                this.updateClientConfigurations();
+                return;
+            }
+        } else {
+            util.addTrustedCompiler(compilerPaths, paths[index]);
+        }
+        // If a compiler is selected, update the default.compilerPath user setting.
+        if (index < paths.length - 3) {
+            settings.defaultCompiler = paths[index];
+        }
+        compilerDefaults = await this.requestCompiler(compilerPaths);
+        this.updateClientConfigurations();
+    }
+
+    async promptSelectCompiler(command: boolean): Promise<void> {
+        const selectCompiler: string = localize("selectCompiler.string", "Select Compiler");
+        const confirmCompiler: string = localize("confirmCompiler.string", "Yes");
+        const settings: OtherSettings = new OtherSettings();
+        if (compilerDefaults.compilerPath !== "") {
+            if (!command) {
+                const value: string | undefined = await vscode.window.showInformationMessage(localize("selectCompiler.message", "The compiler ${0} was found on this computer. Do you want to configure it for IntelliSense?", compilerDefaults.compilerPath), confirmCompiler, selectCompiler);
+                if (value === confirmCompiler) {
+                    compilerPaths = await util.addTrustedCompiler(compilerPaths, compilerDefaults.compilerPath);
+                    settings.defaultCompiler = compilerDefaults.compilerPath;
+                    compilerDefaults = await this.requestCompiler(compilerPaths);
+                    this.updateClientConfigurations();
+                } else if (value === selectCompiler) {
+                    this.handleCompilerQuickPick();
+                } else {
+                    const setCompiler: string = localize("setCompiler.string", "Set Compiler");
+                    const value: string | undefined = await vscode.window.showInformationMessage(localize("setCompiler.message", "You do not have a compiler configured. Unless you set your own configurations, IntelliSense may not be functional."), selectCompiler);
```

Comment:
Same here, it shouldn't say something like "You do not have IntelliSense configured for a compiler."